code/src/App.js

```diff
@@ -1,9 +1,113 @@
-import React from 'react'
+import React, { useState } from "react"
+import data from "./data.json"
+
+import Heading from "./components/Heading"
+import TextInput from "./components/TextInput"
+import SelectInput from "./components/SelectInput"
+import RadioInput from "./components/RadioInput"
+import RangeInput from "./components/RangeInput"
+import CheckboxInput from "./components/CheckboxInput"
+import TextArea from "./components/TextArea"
+import SubmitButton from "components/SubmitButton"
+import SwapCard from "./components/SwapCard"
+import ReturnButton from "./components/ReturnButton"
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
+  const [name, setName] = useState("")
+  const [country, setCountry] = useState("")
+  const [region, setRegion] = useState("")
+  const [unit, setUnit] = useState("")
+  const [amount, setAmount] = useState("")
+  const [allergies, setAllergies] = useState("")
+  const [other, setOther] = useState("")
+  const [summary, setSummary] = useState(false)
+
+  const isRequired = () => {
+    if (name === "") {
+      return false
+    }
+    if (country === "") {
+      return false
+    }
+    if (region === "") {
+      return false
+    }
+    if (unit === "") {
+      return false
+    }
+    if (amount === "") {
+      return false
+    }
+    else {
+      return true
+    }
+  }
+
+  if (summary) {
+    return (
+      <div className="summary">
+        <SwapCard
+          name={name}
+          country={country}
+          region={region}
+          unit={unit}
+          amount={amount}
+          allergies={allergies}
+          other={other}
+        />
+        <ReturnButton
+          setSummary={setSummary}
+        />
+      </div>
+    )
+  } else {
+    return (
+      <div className="form-container">
+        <form>
+          <Heading/>
+          <TextInput
```

Comment:
Using a json-file is probably efficient if you want to make changes but it's a bit harder for me to wrap my head around. Could there be an issue with this method that creates double labels fort the inputs? Beacuse in WAVE I see errors that there are two labels associated with the same input.